package.json

```diff
@@ -1,41 +1,42 @@
 {
-  "license": "ISC",
-  "name": "codesy-widgets",
-  "repository": {
-    "url": "https://github.com/codesy/widgets.git",
-    "type": "git"
-  },
-  "author": "John Dungan",
-  "bugs": {
-    "url": "https://github.com/codesy/widgets/issues"
-  },
-  "version": "1.0.0",
-  "dependencies": {
-    "gulp": "^3.9.0",
-    "gulp-coffee": "2.3.1",
-    "gulp-json-editor": "2.2.1",
-    "gulp-rename": "^1.2.0",
-    "gulp-strip-debug": "1.0.2",
-    "gulp-util": "^3.0.1",
-    "gulp-zip": "2.0.3",
-    "merge-stream": "0.1.7",
-    "gulp-merge-json":"0.5.0"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "keywords": [
-    "github",
-    "codesy",
-    "bid",
-    "fix",
-    "issues"
-  ],
-  "main": "gulpfile.js",
-  "homepage": "https://github.com/codesy/widgets",
-  "id": "jid1-c7AoVz4RhpFuGA",
-  "description": "View codesy bids while on github issue pages",
-  "devDependencies": {
-    "run-sequence": "^1.1.5"
-  }
+    "license": "ISC",
+    "name": "codesy-widgets",
+    "repository": {
+        "url": "https://github.com/codesy/widgets.git",
+        "type": "git"
+    },
+    "author": "John Dungan",
+    "bugs": {
+        "url": "https://github.com/codesy/widgets/issues"
+    },
+    "version": "1.0.0",
+    "dependencies": {
+        "gulp": "^3.9.0",
+        "gulp-coffee": "2.3.1",
+        "gulp-json-editor": "2.2.1",
+        "gulp-rename": "^1.2.0",
+        "gulp-strip-debug": "1.0.2",
+        "gulp-util": "^3.0.1",
+        "gulp-zip": "2.0.3",
+        "merge-stream": "0.1.7",
+        "gulp-merge-json": "0.5.0",
+        "jquery": "3.2.0"
```

Comment:
jquery is also in the static directory.  This just adds a way to get it with `npm install` but you still have to manually copy it from node_modules/jquery/dist.  I'm not ready to commit time to automating that with gulp.